guides/common/modules/proc_configuring-the-discovery-service.adoc

```diff
@@ -37,17 +37,23 @@ endif::[]
 hosts or virtual machines can be created with incorrect network configurations. On Dell servers, the new naming scheme can be disabled via `biosdevname=1` kernel command line option, for other hardware or virtual machines, the new naming scheme can be completely turned off via `net.ifnames=1`.
 endif::[]
 
+[NOTE]
+====
+ifdef::satellite[]
+The Foreman Discovery image provided with {Project} is based on {EL} 8.
+endif::[]
+ifndef::satellite[]
+{Project} provides multiple versions of the Foreman Discovery image (FDI):
+
+* FDI version 4.0 and newer are based on {EL} 8
+* FDI versions older than 4.0 are based on {EL} 7
```

Comment:
Should I be more specific and say [CentOS Stream](https://github.com/theforeman/foreman-discovery-image/blob/master/00-repos-centos8.ks)?